src/osm_configurator/view/popups/alert_pop_up.py

```diff
@@ -1,20 +1,27 @@
 from __future__ import annotations
 
-from customtkinter import CTkToplevel
+import customtkinter
 
+POPUPSIZE = "400x200"  # Holds the size of the Popup
 
-class AlertPopUp(CTkToplevel):
+
+class AlertPopUp(customtkinter.CTk):
     """
     This class creates popups, that will pop up in front of the GUI.
     This instance is an Alert-PopUp. It provides a message and one 'OK' button, to close the PopUp again.
     """
 
     def __init__(self, message):
```

Comment:
typehints missing

Reply:
Good, comment. Will be implemented in the next commit